v2/cycles.json

```diff
@@ -1,184 +1,106 @@
-    "name": "Lunar",
-    "position": 6,
-    "rotated": true,
-    "size": 6
+    "name": "Lunar"
   },
   {
     "code": "order-and-chaos",
-    "name": "Order and Chaos",
-    "position": 7,
-    "rotated": false,
-    "size": 1
+    "name": "Order and Chaos"
   },
   {
     "code": "sansan",
-    "name": "SanSan",
-    "position": 8,
-    "rotated": true,
-    "size": 6
+    "name": "SanSan"
   },
   {
     "code": "data-and-destiny",
-    "name": "Data and Destiny",
-    "position": 9,
-    "rotated": false,
-    "size": 1
+    "name": "Data and Destiny"
   },
   {
     "code": "mumbad",
-    "name": "Mumbad",
-    "position": 10,
-    "rotated": false,
-    "size": 6
+    "name": "Mumbad"
   },
   {
     "code": "flashpoint",
-    "name": "Flashpoint",
-    "position": 11,
-    "rotated": false,
-    "size": 6
+    "name": "Flashpoint"
   },
   {
     "code": "red-sand",
-    "name": "Red Sand",
-    "position": 12,
-    "rotated": false,
-    "size": 6
+    "name": "Red Sand"
   },
   {
     "code": "terminal-directive",
-    "name": "Terminal Directive",
-    "position": 13,
-    "rotated": true,
-    "size": 2
+    "name": "Terminal Directive"
   },
   {
     "code": "core2",
-    "name": "Revised Core Set",
-    "position": 20,
-    "rotated": true,
-    "size": 1
+    "name": "Revised Core Set"
   },
   {
     "code": "kitara",
-    "name": "Kitara",
-    "position": 21,
-    "rotated": false,
-    "size": 6
+    "name": "Kitara"
   },
   {
     "code": "reign-and-reverie",
-    "name": "Reign and Reverie",
-    "position": 22,
-    "rotated": false,
-    "size": 1
+    "name": "Reign and Reverie"
   },
   {
     "code": "magnum-opus",
-    "name": "Magnum Opus",
-    "position": 23,
-    "rotated": false,
-    "size": 1
+    "name": "Magnum Opus"
   },
   {
     "code": "napd",
-    "name": "NAPD Multiplayer",
-    "position": 24,
-    "rotated": false,
-    "size": 1
+    "name": "NAPD Multiplayer"
   },
   {
     "code": "sc19",
-    "name": "System Core 2019",
-    "position": 25,
-    "rotated": true,
-    "size": 1
+    "name": "System Core 2019"
   },
   {
     "code": "ashes",
-    "name": "Ashes",
-    "position": 26,
-    "rotated": false,
-    "size": 3
+    "name": "Ashes"
   },
   {
     "code": "magnum-opus-reprint",
-    "name": "Magnum Opus Reprint",
-    "position": 28,
-    "rotated": false,
-    "size": 1
+    "name": "Magnum Opus Reprint"
   },
   {
     "code": "salvaged-memories",
-    "name": "Salvaged Memories",
-    "position": 29,
-    "rotated": true,
-    "size": 1
+    "name": "Salvaged Memories"
   },
   {
     "code": "system-gateway",
-    "name": "System Gateway",
-    "position": 30,
-    "rotated": false,
-    "size": 1
+    "name": "System Gateway"
   },
   {
     "code": "system-update-2021",
-    "name": "System Update 2021",
-    "position": 31,
-    "rotated": false,
-    "size": 1
+    "name": "System Update 2021"
   },
   {
     "code": "borealis",
-    "name": "Borealis",
-    "position": 32,
```

Comment:
You've no doubt discussed this on discord but mind explaining why you're removing this info?

Reply:
That all makes sense to me. Thanks for the explanation.